praetorian_cli/sdk/test/ui/test_console_data.py

```
@@ -0,0 +1,299 @@
+        csv_content = "dns,type,status\nexample.com,asset,A\ntest.org,asset,P\n"
+        with tempfile.NamedTemporaryFile(suffix='.csv', mode='w', delete=False) as f:
+            f.write(csv_content)
+            tmp_path = f.name
+        try:
+            cmd._cmd_import(['seeds', tmp_path])
+            assert len(cmd.sdk.seeds.calls) == 2
+            assert cmd.sdk.seeds.calls[0]['dns'] == 'example.com'
+            assert cmd.sdk.seeds.calls[0]['seed_type'] == 'asset'
+            assert cmd.sdk.seeds.calls[0]['status'] == 'A'
+            assert cmd.sdk.seeds.calls[1]['dns'] == 'test.org'
+            assert any('2' in line and 'success' in line.lower() or 'added' in line.lower() for line in cmd.console.lines)
```

Ruff (0.15.9)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)